### ISSUE TITLE
llvd failed to download a number of videos - LinkedIn Learning restriction?

### ISSUE THREAD

**Issue by michael-hart-github** [knowbee/llvd#77]
**URL**
`https://www.linkedin.com/learning/linux-tips-weekly/`

**Command run**
`llvd -c "linux-tips-weekly" -t 5,10`

**Errors observed**
> Failed to download Migrating to a new distro
> Failed to download Planning a portable bootable disk
> Failed to download Creating a portable bootable disk
> Failed to download Host your own Apache web server
> Failed to download Set up Let’s Encrypt on a web server
> Failed to download Host a personal cloud with Nextcloud
> Failed to download Hosting email services
> Failed to download Set up a print server with CUPS
> Failed to download Create a network PDF printer
> Failed to download Remote logging with Rsyslog
> Failed to download Set up a Git server
> Failed to download NGINX web server
> Failed to download Share files with a Samba server
> Failed to download Set up an NFS server
> Failed to download Monitoring your server
> Failed to download Exploring Internet Relay Chat (IRC)

The full output is rather long, I've attached it in a text file.
[llvd-error.txt](https://github.com/knowbee/llvd/files/8147391/llvd-error.txt)

**Attempted solutions**
Re-run the command, increase the throttle.
`llvd -c "linux-tips-weekly" -t 30,120`

**Results**
I'm still seeing errors. It looks like each time I run the command, _some_ of the videos `llvd` missed on previous attempts is now able to be downloaded. Strange.
[llvd-error-2.txt](https://github.com/knowbee/llvd/files/8147402/llvd-error-2.txt)

I'm not sure if this is a problem with `llvd`, or LinkedIn Learning itself. I suspect it might be a problem with LinkedIn Learning (maybe content was removed by the author, but perhaps `llvd` still detects it?) Whatever the case, I thought it would be helpful to report.

Cheers. Love the software! :)

**Comment by michael-hart-github** [comment]
I've tried the exact same command a few hours later
` llvd -c "linux-tips-weekly" -t 30,120` and it seems to have worked perfectly. Either there was some sort of issue with my account (temp banned?) or some type of problem with the website.

I had noticed that when I tried visiting the site, the logo would show in the upper left, and the language option in the upper right, but the rest of the screen was completely blank. Weird. :man_shrugging: :smile:
